components/CalendarEvents.js

```diff
@@ -2,7 +2,7 @@ import React, { useEffect, useState } from "react";
 import moment from "moment";
 import { Calendar, momentLocalizer } from "react-big-calendar";
 import "react-big-calendar/lib/css/react-big-calendar.css";
-import { Events } from "./Events.js";
+import { Events } from "./data/Events.js";
```

Comment:
if it is default exported, you can just do
```suggestion
import Events from "./data/Events.js";
```

Reply:
done to all of them (/data modules use default export  now)